exercises/practice/queen-attack/src/test/java/QueenAttackCalculatorTest.java

```diff
@@ -109,6 +122,8 @@ public void testQueensCanAttackOnFourthDiagonal() {
 
     @Disabled("Remove to run test")
     @Test
+    @DisplayName("cannot attack if falling diagonals are only the same " +
+            "when reflected across the longest falling diagonal")
```

Comment:
```suggestion
    @DisplayName(
        "cannot attack if falling diagonals are only the same " +
        "when reflected across the longest falling diagonal"
    )
```

Just this small change to make it consistent with the earlier files!